src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java

```diff
@@ -46,7 +46,7 @@ void setUp() {
 
     @Test
     void whenMeshInboundQueueMessageIsReceivedThenMessageIsHandled(SoftAssertions softly)
-            throws IOException, JMSException, JSONException {
+        throws IOException, JMSException, JSONException {
```

Comment:
I'd revert this change, this part of the method header should be indented so it's clear from the method body. Think this may be happening due to the new formatting rules we've all imported to IntelliJ